examples/example-jwt-auth/src/main/java/io/grpc/examples/jwtauth/BearerToken.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.examples.jwtauth;
+
+import io.grpc.CallCredentials;
+import io.grpc.Metadata;
+import io.grpc.Status;
+import java.util.concurrent.Executor;
+
+/**
+ * CallCredentials implementation, which carries the JWT value that will be propagated to the server
+ * in the request metadata with the "Authorization" key and the "Bearer" prefix
+ */
+public class BearerToken extends CallCredentials {
```

Comment:
This class is only used in the client - one way conversion from JWT to bearer token but not on the server side. 

- can we see if it can be used in both places
- if not - make it private for the client?

Reply:
I don’t see how we could use it on the server side, since it only used to add `Metadata` to the request, and on the server, we only need the `Metadata` key to get the value. Perhaps we could encapsulate the constant with the `Metadata` key in this class and use it on the server only to get this constant, but since we have a separate class for the constants, I prefer to leave it there.

By making it private for the client, do you mean splitting the client and server into separate packages, or making it an internal class? I would prefer to leave it as it is, so as not to complicate the understanding of the example.